e2e/wdio/core/fixtures/testData/select-tags.ts

```diff
@@ -0,0 +1,38 @@
+const prefix = 'select-core-';
+export const selectModeExample = prefix + 'select-mode-example-';
+export const semanticExample = prefix + 'semantic-example-';
+export const customControlExample = prefix + 'custom-control-example-';
+export const extendedExample = prefix + 'extended-example-';
+export const mobileExample = prefix + 'mobile-example-';
+export const maxHeightExample = prefix + 'max-height-example-';
+export const addRemoveExample = prefix + 'add-remove-example-';
+export const programmaticExample = prefix + 'programmatic-example-';
+
+export const optionTag = 'option-';
+
```

Comment:
please remove any unneeded empty rows from the file

Reply:
updated